component/commons/handler/src/main/java/org/wso2/carbon/identity/password/expiry/validation/handler/PasswordExpiryPolicyUtils.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.password.expiry.validation.handler;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+import org.wso2.carbon.identity.application.common.model.IdentityProviderProperty;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationManagementUtil;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManager;
+
+/**
+ * Read password expiry in days configuration.
+ */
+public class PasswordExpiryPolicyUtils {
+    private static final Log log = LogFactory.getLog(PasswordExpiryPolicyUtils.class);
+
+    private PasswordExpiryPolicyUtils(){}
+
+    /**
+     * Get identity event property for password expiry in days.
+     * @param tenantDomain user tenant domain
+     * @param propertyName
+     * @return property value
+     */
+    public static String getIdentityEventProperty(String tenantDomain, String propertyName) {
```

Comment:
I feel unnesarry config readings..  We beter write the logic in passwordExpiry and get rid of these utils.